binary-serialise-cbor/cborg-json/src/Codec/CBOR/JSON.hs

```diff
@@ -0,0 +1,98 @@
+-- |
+-- Module      : Codec.CBOR.JSON
+-- Copyright   : (c) Duncan Coutts 2015-2017
+-- License     : BSD3-style (see LICENSE.txt)
+--
+-- Maintainer  : duncan@community.haskell.org
+-- Stability   : experimental
+-- Portability : non-portable (GHC extensions)
+--
+-- Encoding JSON values in CBOR.
+--
+module Codec.CBOR.JSON
+ ( encodeValue
+ , decodeValue
+ ) where
+
+import           Data.Aeson                          ( Value(..) )
+import qualified Data.Aeson                          as Aeson
+import           Data.Aeson.Encode.Pretty            as Aeson.Pretty
```

Comment:
What do we use the `Aeson.Pretty` for here? I thought it was only in the cbor-tool for pretty-printing json output.